app/views/admin/profile_fields/_profile_field_form.html.erb

```diff
@@ -1,3 +1,14 @@
+<div class="form-group">
+  <%= form.label :group %>
+  <%= select_tag "profile_field[profile_field_group_id]",
+                 options_from_collection_for_select(
+                   ProfileFieldGroup.all,
+                   :id, :name,
+                   selected: !group ? nil : group.id
```

Comment:
**suggestion (non-blocking):** I think `gropu&.id` or `group.try(:id)` would be easier to read than this ternary.

Reply:
I always forget ruby has this neat trick. Thanks!